handbook/ce/onboarding.md

```diff
@@ -21,14 +22,18 @@ Welcome to Sourcegraph! This document will guide you through customer engineerin
 - [Install Sourcegraph](https://docs.sourcegraph.com/admin/install/) using each of the three deployment options
   - Ask for any help needed along the way in the #dev-chat channel
   - Update our docs where necessary
-- Participate in (or be scheduled for) the following training sessions. Please ask your manager or People Ops to help you find the right training sessions to join or the right training videos to watch.
-  - Goals (@dan, 30 minutes)
-  - CE systems and processes (@dan, 1 hour)
-  - Product introduction (@christina, 1 hour)
+- Participate in (or watch the recording of) the following training sessions. Please ask your manager or People Ops to help you find the right training sessions to join if you cannot find the right training videos to watch.
+  - CE Goals
+  - CE systems and processes 
+  - Product introduction 
     - Optionally request access to [Figma](https://www.figma.com/) in the #product channel on Slack
-  - Sourcegraph demo (@dan, 30 minutes)
-  - Product deployment (@stephen, 30 minutes)
-  - Sourcegraph data (@ericbm, 30 minutes)
+  - Sourcegraph demo 
+  - Product deployment 
+  - Sourcegraph data 
+- Go through [Product](https://about.sourcegraph.com/handbook/product/onboarding) and [Engineering](https://about.sourcegraph.com/handbook/engineering/onboarding) onboarding, specifically to gain understanding of:
+  - How our product is architected, how it works together, etc. [Sourcegraph architecture](https://docs.sourcegraph.com/dev/background-information/architecture), and all the resources on that page, are a very helpful read-through (as well as a bookmark to come back to later).
+  - How product is developed here.
+  - Other ways other team members are onboarded.
 - Set up introductory calls with Distribution, Sales, Product, and Marketing team members
```

Comment:
I would like to avoid encoding that the distribution team is special in any way.

```suggestion
- Set up introductory calls with Sales, Product, and Marketing team members
```